### PR TITLE
Subscribers end point

### PR DESCRIPTION
**Pull request by akintolga**
[SD-2048] - Subscribers end point created
[SD-2047] - Output channels are changed to have list of subscribers

### CODE REVIEW

**Comment by marwoodandrew** [comment]
:+1:

File: server/features/subscribers.feature [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+      "name":"Channel 3"
+    }
+    """
+    Then we get latest
+    """
+    {
+      "name":"Channel 3"
+    }
+    """
+    When we post to "/output_channels"
```
Comment (amagdas) [inline server/features/subscribers.feature]: We should check that the post works as expected. We can do it by calling:
`When we post to "/output_channels" with success`

**Comment by mdhaman** [comment]
:+1:  :100: 
I had to comment on this ....

**Comment by amagdas** [comment]
:+1: 
@mdhaman stay away from the computer :smile:

**Pull request merged**